modules/face/samples/sample_config_file.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
```

Comment:
I have added the comments in the sample .cpp file writing this file. It explains the use of each parameter.